lib/perl/Genome/Sample/Command/Import/Base.pm

```diff
@@ -258,9 +248,11 @@ sub _create_individual {
     return $self->_individual($individual);
 }
 
-sub _create_sample {
+sub _create_sample_if_needed {
     my $self = shift;
 
+    return 1 if $self->_sample;
```

Comment:
Seems like the initial setting of `_sample`, currently in `_does_library_already_exist`, could be moved here?  If so then `_does_library_already_exist` is basically `_get_library` so `_does_library_already_exist` could be removed.